### ISSUE TITLE
GIOU Training Loss

### ISSUE THREAD

**Issue by os1a** [generalized-iou/g-tensorflow-models#5]
Hi,
I am trying to train a simple network which outputs a single bounding box against ground-truth.
Using the smooth L1 loss, it worked.
When using the GIOU loss, it get stucked in producing non-sense bounding boxes (coordinates are too large) and the loss stays always fixed (1.0)
After debugging, it seems that the network directly after few iterations pushs the giou score to be zero and thus the loss stays always 1.

Could you please provide more help in this regard?
Best,
Osama
